lib/charms/mysql/v0/backups.py

```diff
@@ -98,10 +99,14 @@ def is_unit_blocked(self) -> bool:
 LIBPATCH = 6
```

Comment:
would we need to update the libpatch?

Reply:
Given that there's no functional changes (just typing) it does not seems right

Reply:
i suggested the libpatch update so that we can easily port improvements over to the k8s charm (else these changes may get lost at some point)